eos-enable-zram

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env sh
```

Comment:
Below you're using some bash features, so I suggest just having this be `#!/bin/bash`. It might also be a good idea to add `-e` to exit on errors. E.g., if you can't modprobe zram, then you probably shouldn't keep going.